customizations/README.md

```diff
@@ -1042,6 +1043,14 @@ Aligned with the [high availability configurations](high_availability), please c
 - [Standard HA with upgrades](high_availability/standard_with_upgrades/README.md#capacity-planning)
 - [High traffic HA](high_availability/high_traffic/README.md#capacity-planning)
 
+## Pod Disruption Budget
+
+Along with the HA configurations, we can use `podDisruptionBudget` Kubernetes feature to define the budget of voluntary disruption by letting the cluster aware of a minimum threshold in terms of available pods that the cluster needs to guarantee in order to ensure a baseline availability or performance.
```

Comment:
I think it's better `making the cluster aware of` .vs. `letting the cluster aware of`.